client/src/routes/screens/Sessions/Sessions.tsx

```diff
@@ -0,0 +1,140 @@
+import {Heading16} from '../../../lib/components/Typography/Heading/Heading';
+import {SPACINGS} from '../../../lib/constants/spacings';
+import useExerciseById from '../../../lib/content/hooks/useExerciseById';
+import useExerciseIds from '../../../lib/content/hooks/useExerciseIds';
+import {
+  ModalStackProps,
+  OverlayStackProps,
+} from '../../../lib/navigation/constants/routes';
+import {formatExerciseName} from '../../../lib/utils/string';
+
+const BottomGradient = styled(LinearGradient)({
+  position: 'absolute',
+  left: 0,
+  right: 0,
+  bottom: 0,
+  height: 40,
+});
+
+type ExerciseCardProps = {
+  exerciseId: string;
+  hasCardBefore: boolean;
+  hasCardAfter: boolean;
+};
+const ExerciseCard: React.FC<ExerciseCardProps> = ({
+  exerciseId,
+  hasCardBefore,
+  hasCardAfter,
+}) => {
+  const {navigate} =
+    useNavigation<NativeStackNavigationProp<ModalStackProps>>();
+  const exercise = useExerciseById(exerciseId);
+
+  const image = useMemo(() => {
+    if (exercise?.card?.image) {
+      return {uri: exercise.card.image.source};
+    }
+  }, [exercise]);
+
+  const onPress = useCallback(() => {
+    navigate('CreateSessionModal', {exerciseId, discover: true});
+  }, [exerciseId, navigate]);
+
+  if (!exercise) {
+    return null;
+  }
+
+  return (
+    <ExerciseWalletCard
+      title={formatExerciseName(exercise)}
+      image={image}
+      hasCardBefore={hasCardBefore}
+      hasCardAfter={hasCardAfter}
+      onPress={onPress}
+    />
+  );
+};
+
+const Sessions = () => {
+  const {navigate} =
+    useNavigation<NativeStackNavigationProp<OverlayStackProps>>();
+  const exerciseIds = useExerciseIds();
+  const {t} = useTranslation('Screen.Sessions');
+  const colors = useMemo(
+    () => [hexToRgba(COLORS.WHITE, 0), hexToRgba(COLORS.WHITE, 1)],
+    [],
+  );
+
+  const onPressEllipsis = useCallback(() => {
+    navigate('AboutOverlay');
+  }, [navigate]);
+
+  const renderItem = useCallback<ListRenderItem<string>>(
+    ({item, index}) => {
+      const hasCardBefore = index > 0;
+      const hasCardAfter = index !== exerciseIds.length - 1;
+      return (
+        <Gutters>
+          <ExerciseCard
+            exerciseId={item}
+            hasCardBefore={hasCardBefore}
+            hasCardAfter={hasCardAfter}
+          />
+        </Gutters>
+      );
+    },
+    [exerciseIds],
+  );
+
+  const keyExtractor = useCallback((id: string) => id, []);
+
+  return (
+    <Screen backgroundColor={COLORS.PURE_WHITE}>
+      <TopSafeArea minSize={SPACINGS.SIXTEEN} />
+      <TopBar
+        backgroundColor={COLORS.PURE_WHITE}
+        onPressEllipsis={onPressEllipsis}>
+        <MiniProfile />
+      </TopBar>
+      <Spacer16 />
+
+      <FlatList
+        data={exerciseIds}
+        renderItem={renderItem}
+        keyExtractor={keyExtractor}
+        ListHeaderComponent={
+          <Gutters>
+            <Heading16>{t('sessionsHeading')}</Heading16>
+            <Spacer8 />
+          </Gutters>
+        }
```

Comment:
Wouldn't it be good to make this sticky already now? And align it with `Home` and `Journey`.